exercises/exercise_2.js

```diff
@@ -2,6 +2,7 @@ const characters = require("../data/characters.json");
 
 // TODO: return all characters with known birth year
 const charactersWithKnownBirthDate = () => {
+  return characters.filter(character => character.birth_year !== "unknown")
```

Comment:
Great using !== instead of !=

Great use of find and filter, of arrow function (ES6...) 👏 👏 👏 👏 👏 👏 👏